src/olympia/files/models.py

```diff
@@ -706,6 +712,50 @@ class Meta:
         db_table = 'validation_annotations'
 
 
+class WebextPermission(ModelBase):
+    KNOWN_PERMISSIONS = WEBEXT_PERMISSIONS_DICT
+    name = models.CharField(max_length=255, unique=True, null=False)
+    _enum = models.IntegerField(default=WEBEXT_UNKNOWN.id, db_column='enum')
+    files = models.ManyToManyField(File, related_name='webext_permissions',
+                                   through='FileWebextPermission')
```

Comment:
nit: Again, not a fan of the `ManyToManyField` approach but I'd make more sense to have this on the `File` class if we do that